libs/NfcCoreLib/lib/Fri/phFriNfc_ISO15693Map.c

```diff
@@ -370,6 +363,17 @@ phFriNfc_ISO15693_H_ReadWrite (
     {
         request_flags |= ISO15693_FLAG_PROTOEXT;
     }
+    else if (ps_iso_15693_con->support_extended_cmd == TRUE)
+    {
+        /* Force extended command */
+        command |= ISO15693_EXTENDED_CMD_MASK;
+    }
+
+    if (ISO15693_READ_MULTIPLE_COMMAND == command)
+    {
+        /* Force data_length to 1 as we are using ISO15693_READ_MULTIPLE_COMMAND */
+        data_length = 1;
```

Comment:
Why is this set to 1?

Reply:
I think there is a coding mistake as well here visible from phFriNfc_ReadRemainingInMultiple where its seems the wrong command is used depending of the different checks.
I will try to come with a fix.